core/store/lib/search/adapter/graphql/queries/cmsPage.gql

```diff
@@ -0,0 +1,14 @@
+query cmsPages ($filter: CmsInput) {
```

Comment:
should be in module/queries

Reply:
This gql is related to the certain searchAdapter. Not sure if it should be in the module/queries

Reply:
Same as above

Reply:
okay ;)